iOS35/Presentation/Main/App/Financial/Cell/RankingTableViewCell.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  RankingTableViewCell.swift
+//  iOS35
+//
+//  Created by 우상욱 on 11/1/24.
+//
+
+import UIKit
+import SnapKit
+import Then
+protocol RankingTableViewCellDelegate: AnyObject {
+    func rankingTableViewCellTapped(title: String)
+}
+class RankingTableViewCell: UITableViewCell {
+    weak var delegate: RankingTableViewCellDelegate?
+    
+    private let appIcon = UIImageView().then {
+        $0.contentMode = .scaleAspectFill
+        $0.clipsToBounds = true
+        $0.layer.cornerRadius = 12
+    }
+    
+    private let rankLabel = UILabel().then {
+        $0.font = .systemFont(ofSize: 18)
+        $0.textColor = .white
+    }
+    
+    private let appTitle = UILabel().then {
+        $0.font = .systemFont(ofSize: 18)
+        $0.textColor = .white
+    }
+    
+    private let appSubTitle = UILabel().then {
+        $0.font = .systemFont(ofSize: 14)
+        $0.textColor = .gray
+    }
+    
+    private let downloadButton = UIButton().then {
+        $0.setTitle("받기", for: .normal)
+        $0.setTitleColor(.tintColor, for: .normal)
+        $0.titleLabel?.font = .systemFont(ofSize: 18)
+        $0.layer.cornerRadius = 15
+        $0.backgroundColor = UIColor(white: 0.1, alpha: 1)
+    }
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        self.backgroundColor = .black
+        self.selectionStyle = .none
+        
+        addSubviews()
+        setupLayout()
+        setupEvent()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension RankingTableViewCell {
+    
+    private func addSubviews() {
+        [appIcon, rankLabel, appTitle, appSubTitle, downloadButton].forEach { [weak self] view in
+            guard let self else { return }
+            contentView.addSubview(view)
+        }
+    }
+    
+    private func setupLayout() {
+        appIcon.snp.makeConstraints {
+            $0.leading.equalToSuperview()
+            $0.width.height.equalTo(60)
+            $0.top.bottom.equalToSuperview().inset(10)
```

Comment:
```suggestion
           $0.verticalEdges.equalToSuperview().inset(10)
```
top과 bottom에 같은 여백 값을 줄 때, verticalEdges로 한번에 표현할 수 있습니다. 
leading과 trailing에 같은 값을 줄 땐 horizontalEdges로 표현할 수 있겠죠?